.travis.yml

```diff
@@ -142,22 +115,6 @@ jobs:
     - <<: *STANDARD_TEST_JOB
       env: MODULE='cql' ARGS='-Pcassandra2-murmur -Dtest=**/hadoop/*'
 
-    - <<: *FULL_BUILD_JOB
-      env: MODULE='cql' ARGS='-Dtest=**/graphdb/cql/* -Dtest.skip.byteorderedpartitioner=true -Dtest.skip.murmur-serial=true -Dtest.skip.murmur-ssl=true'
```

Comment:
This test makes no sense. We did this already.

Reply:
I see there are two graphdb/cql tests for cql module:
```
    - <<: *STANDARD_TEST_JOB
      env: MODULE='cql' ARGS='-Pcassandra2-byteordered -Dtest=**/graphdb/cql/*'
    - <<: *STANDARD_TEST_JOB
      env: MODULE='cql' ARGS='-Pcassandra2-murmur -Dtest=**/graphdb/cql/*'
```
Does it mean that we don't need this test because we have two of above tests? Or does it mean that this test is redundant due to this PR (removal of Thrift)?
If this change isn't related to this PR, may we just move it to a separate PR?

Reply:
This change is not related to this PR.